lib/controllers.ts

```diff
@@ -0,0 +1,33 @@
+// export all controllers *with helpers* so they can be bulk re-exported by the package entry point
```

Comment:
I am a bit unsure about the location of this file. Conceptually to me it is not too different from something like the `stacks-dynamic.less` and that resides directly in the `lib` folder. So maybe this one should too for the sake of consistency? Thoughts?

Reply:
This file _does_ reside directly in the `lib` folder. Am I misunderstanding what your suggestion?